Assets/Tests/PlayMode/EOSConfigurationTests.cs

```diff
@@ -0,0 +1,257 @@
+﻿/*
+ * Copyright (c) 2024 PlayEveryWare
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+namespace PlayEveryWare.EpicOnlineServices.Tests.IntegrationTests
+{
+    using Epic.OnlineServices;
+    using Epic.OnlineServices.Sessions;
+    using NUnit.Framework;
+    using EpicOnlineServices;
+    using System.Collections;
+    using UnityEngine;
+    using UnityEngine.TestTools;
+    using Epic.OnlineServices.Friends;
+    using Epic.OnlineServices.TitleStorage;
+    using Epic.OnlineServices.Platform;
+    using PlayEveryWare.EpicOnlineServices.Tests;
+    using static PlayEveryWare.EpicOnlineServices.EOSManager;
+
+    /// <summary>
+    /// Tests for values set in <see cref="EOSConfig"/> and how they interact with the EOS SDK.
+    /// This test repeatedly sets up and tears down the EOS Libraries, so it doesn't inherit from the <see cref="EOSTestBase"/>.
+    /// </summary>
+    public class EOSConfigurationTests
```

Comment:
This class should inherit from `EOSTestBase` - as that class contains a lot of the functionality that is common to all tests (for example, this is why the `TestCategories` class was made to be `protected`, because it should only be utilized within classes that extend from `EOSTestBase`).

Reply:
I disagree here. EOSTestBase authenticates and logs in to Epic automatically. I _don't_ want that to happen, I need the specific state management in order to load and unload the SDK.